src/CLITests/ErrorCollectorUnitTests.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using AxeWindowsScanner;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+
+namespace CLITests
+{
+    [TestClass]
+    public class ErrorCollectorUnitTests
+    {
+        const string TrivialValue = "   \n   \n\t\n";
+        const string ValidValue = "Some\n Valid\n Value";
+        static readonly string[] ValidValueArray = { "Value 1", "Value 2", "Value 3" };
+
+        IErrorCollector _errorCollector;
+
+        [TestInitialize]
+        public void BeforeEachTest()
+        {
+            _errorCollector = new ErrorCollector();
+        }
+
+        private void ValidateListSizes(int expectedParameterErrors = 0, int expectedExceptions = 0)
```

Comment:
I'm not a fan of default parameter values because it requires me to remembe what the defaults are, or that there _are_ defaults at all. Even after reading this function, I still found it to be a speed bump when I encountered the method call with no parameters.

Reply:
This class is going away, but I find defaults to builders or validators extremely useful in unit tests